packages/app/src/cli/services/flow/validation.ts

```diff
@@ -34,17 +35,13 @@ export const validateFieldShape = (
           required: zod.boolean().optional(),
         })
         .parse(configField)
+    } else {
+      return FlowTriggerSettingsSchema.parse(configField)
     }
-
-    return baseFieldSchema
-      .extend({
-        key: zod.string(fieldValidationErrorMessage('key', configField, extensionHandle, index)),
-      })
-      .parse(configField)
   }
 
   if (isCommerceObjectField) {
-    if (type === 'flow_action') {
+    if (type === 'flow_action' || type === 'flow_trigger') {
```

Comment:
Do need this `if` at all anymore? when will this evaluate `false`?

Reply:
I guess it won't, I can remove it